packages/react-components/src/components/VideoGallery/FloatingLocalVideoLayout.tsx

```diff
@@ -51,23 +51,25 @@ export const FloatingLocalVideoLayout = (props: FloatingLocalVideoLayoutProps):
     maxRemoteVideoStreams,
     showCameraSwitcherInLocalPreview,
     parentWidth,
-    parentHeight
+    parentHeight,
+    /* @conditional-compile-remove(pinned-participants) */ pinnedParticipantUserIds
   } = props;
 
   const theme = useTheme();
 
   const isNarrow = parentWidth ? isNarrowWidth(parentWidth) : false;
 
-  const floatingLocalVideoLayout = useFloatingLocalVideoLayout({
+  const layout = useLayout({
```

Comment:
nit: `useLayout` is a vague (hard to know what a `layout` entails, I had to go look at the function definition to understand what this hook did), consider something like:
```suggestion
  const { gridParticipants, horizontalGalleryParticipants } = useOrganizedParticipants({
```

(struggling also to come up with a better word, not in love with `useOrganizedParticipants` but I think just having the return value immediately split goes a long way in understanding what this function does)